spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-configuration-watcher-it/pom.xml

```diff
@@ -53,45 +53,25 @@
 			<groupId>io.kubernetes</groupId>
```

Comment:
we were missing logs here, so I just cleaned up things a little for them to be present